apps/web-giddh/src/app/new-inventory/component/adjust-inventory/adjust-inventory.component.html

```diff
@@ -0,0 +1,407 @@
+                        <reactive-dropdown-field
+                            [label]="commonLocaleData?.app_import_type?.inventory"
+                            [placeholder]="commonLocaleData?.app_select_inventory_type"
+                            [options]="inventoryList$ | async"
+                            [labelValue]="adjustInventoryCreateEditForm.get('entityName')?.value"
+                            formControlName="entityUniqueName"
+                            [required]="true"
+                            (selectedOption)="selectInventory($event)"
+                            (onClear)="adjustInventoryCreateEditForm.get('entityUniqueName')?.patchValue(null)"
+                            [showError]="
+                                isFormSubmitted && !adjustInventoryCreateEditForm.get('entityUniqueName')?.value
+                            "
+                        >
+                        </reactive-dropdown-field>
+                    </div>
+                    <div class="col-xxl-3 mt-3 col-xl-2 col-lg-6 col-md-6" *ngIf="showHideTable">
+                        <reactive-dropdown-field
+                            [label]="localeData?.adjustment_method"
+                            [placeholder]="commonLocaleData?.app_select + ' ' + localeData?.adjustment_method"
+                            [options]="adjustmentMethod"
+                            [labelValue]="adjustInventoryCreateEditForm.get('adjustmentMethodName')?.value"
+                            formControlName="adjustmentMethod"
+                            [required]="true"
+                            [readonly]="allApiCalled"
+                            (selectedOption)="
+                                calculateCalculation(adjustInventoryCreateEditForm, stockGroupClosingBalance)
+                            "
+                            (onClear)="adjustInventoryCreateEditForm.get('adjustmentMethod')?.patchValue(null)"
+                            [showError]="
+                                isFormSubmitted && !adjustInventoryCreateEditForm.get('adjustmentMethod')?.value
+                            "
+                        >
+                        </reactive-dropdown-field>
+                    </div>
+                    <div class="col-xxl-3 mt-3 col-xl-2 col-lg-6 col-md-6">
+                        <reactive-dropdown-field
+                            [label]="localeData?.reason"
+                            [placeholder]="commonLocaleData?.app_select + ' ' + localeData?.reason"
+                            [options]="reasons$ | async"
+                            formControlName="reasonUniqueName"
+                            [labelValue]="adjustInventoryCreateEditForm.get('reasonName')?.value"
+                            [required]="true"
+                            (createOption)="openCreateReasonModal()"
+                            [createNewText]="commonLocaleData?.app_create + ' ' + localeData?.reason"
```

Comment:
this will create translation issue in hindi/marathi

Reply:
Done